pkg/operator/sync.go

```diff
@@ -622,6 +625,11 @@ func (optr *Operator) syncCustomResourceDefinitions() error {
 	}
 
 	for _, crd := range crds {
+		DNE := false
```

Comment:
Suggestion: Make `DNE` lowercase, e.g., `dne`.

Variables beginning with an uppercase letter in Go are exported from a given module. While this variable would not be exported via that mechanism, it does look a bit out of place here.